### ISSUE TITLE
MatCheckedBox @OnChanged NotWork

### ISSUE THREAD

**Issue by HossienMehdikhah** [SamProf/MatBlazor#757]
**Describe the bug**
hi,
I Want Run A method when a MatCheckedBox Checked Or Uncheked.
in Blazor i Use <input> and Put my Method in OnChange <input @onChange="Temp",

private void Temp(EventChangeArg e)
{
......
}
and Run My Method
but in Matblazor dont Work.

**Comment by Christian-Oleson** [comment]
@HossienMehdikhah ,

Thanks for opening an issue. Would you mind providing a Blazor fiddle example as well as following the bug guidelines and reporting?
Thanks!

**Comment by khew81** [comment]
I am meeting the same problem too. The correct way shall be  <input @onchange="@Temp">, you're missing the **@**. I am meeting the same bug too.

**Comment by Christian-Oleson** [comment]
@khew81 ,
What error occurs?

**Comment by khew81** [comment]
@Christian-Oleson No errors but the event just not being triggered, after I changed from <MatSelectItem @onchange="@OnChangeEvent> to <Select @onchange="@OnChangeEvent>the on change event triggering just works. I'm currently using the Bootstrap <Select> until it is fixed or if MatBlazor requires different style of doing it please help to guide me.

**Comment by HossienMehdikhah** [comment]
i find That i read .net5. it say that this ability provide in .net5 not .net core 3.1.
so i will wait to release .net5

**Comment by khew81** [comment]
> i find That i read .net5. it say that this ability provide in .net5 not .net core 3.1.
> so i will wait to release .net5

What is your temporary solution for this?

**Comment by HossienMehdikhah** [comment]
> > i find That i read .net5. it say that this ability provide in .net5 not .net core 3.1.
> > so i will wait to release .net5
> 
> What is your temporary solution for this?

truly. i Replace matblazor with devexpress blazor freamwork.

**Comment by khew81** [comment]
> > > i find That i read .net5. it say that this ability provide in .net5 not .net core 3.1.
> > > so i will wait to release .net5
> > 
> > 
> > What is your temporary solution for this?
> 
> truly. i Replace matblazor with devexpress blazor freamwork.

I see you completely replaced matblazor with devexpress blazor freamwork. You can check up Ant Design Blazor as an alternative too.

**Comment by HossienMehdikhah** [comment]
> Ant Design Blazor

yes .its Good Framework . but i use codeRush so i like DevExpress and their components is excellent and coverage Deference needs and situation
